### ISSUE TITLE
Website: Add Platform-Specific Tags on styling properties

### ISSUE THREAD

**Issue by mathdebate09** [nativeflowteam/documentation#3]
### Checks

- [x] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/nativeflowteam/documentation/issues) to make sure)
- [x] I have thoroughly read and understand the [Contributing Guide](https://github.com/nativeflowteam/documentation/blob/main/CONTRIBUTING.md)
- [x] Would you like to work on this issue?

### Why is this needed?

To make the documentation more concise, which also acts as awarning for the user that this porperty will be applicable only on certain OS

### Guidelines

- Comment on this issue to let us know you'd like to get a section assigned to you. You can also ask for a specific one.
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons.
- When you have been assigned a section, please refer to the [suggested link text changes spreadsheet](https://docs.google.com/spreadsheets/d/1p3UktA7lbAf-wSvp1NiZPgbJuOZwOtbQUJsGZmSXkXs/edit?usp=sharing) for instructions on what changes need to be made and checkout [where to edit guide below](#where-to-edit-guide) for locating your changes
   - For most of these, you should just be able to copy and paste the suggested text to replace the original text shown in the spreadsheet. Any additional steps/clarifications will be noted by the appropriate link(s).
  - You are welcome to use a different wording from a given suggestion if you find a better one. If you do, you must ensure the link text is consistent if there are multiple instances of the same href in the same lesson.
  - You are not required to fix any unrelated, preexisting linting errors such as list styles/indentation. Existing linting errors may cause certain tests to fail when you open a PR. You can ignore this and a maintainer will let you know if further action is required.
- Make sure to read the [general contributing guide](https://github.com/nativeflowteam/.github/blob/main/CONTRIBUTING.md) as well as the [curriculum repo contributing guide](https://github.com/nativeflowteam/documentation/blob/main/CONTRIBUTING.md) before starting your first contribution.
- Please add the following to your PR in the Issues section: **Related to #28290**. Since this issue is divided into sections, individual PRs should **NOT INCLUDE** the **closes** keyword next to an issue number.

### Where to edit Guide

Spreadsheet will provide you location of which file to change, you have to add the symbol in the heading front matter in the `mdx`, like below example

docs/typography/text-decoration-style.mdx

```md
---
id: text-decoration-style
title: Text Decoration Style ★ # add symbol here
description: Define the style of text decorations, like solid or dotted.
slug: /text-decoration-style
sidebar_position: 8
---

# ...
# ...
# add docs reference at the end

>Reference: [textDecorationStyle | React Native](https://reactnative.dev/docs/text-style-props#textdecorationstyle-ios)

```

### Progress & Assignees

- [x] @mathdebate09 [Typography](https://nativeflow.js.org/docs/typography)
- [x] @mathdebate09 [Shadow](https://nativeflow.js.org/docs/shadow)
- [x] @mathdebate09 [Effect](https://nativeflow.js.org/docs/effects)

### (Optional) Discord username

*Optional*
